OnlineUIComponents/ChatHud.cs

```diff
@@ -13,8 +13,8 @@ public class ChatHud : HudPart
         private ChatLogOverlay? chatLogOverlay;
         private ChatInputOverlay? chatInputOverlay;
         public bool chatInputActive => chatInputOverlay is not null;
-        private bool showChatLog = false;
-        
+        public bool showChatLog = RainMeadow.rainMeadowOptions.ChatLogOnOff.Value;
```

Comment:
consider making this a property that sets the configurable Value directly so users can toggle persist in-game as well